TerpstraSysEx.jucer

```diff
@@ -303,9 +303,9 @@
                 linuxExtraPkgConfig="libssh2" extraDefs="HAVE_ARPA_INET_H=1 HAVE_UNISTD_H=1 HAVE_POLL_H=1 SERVERHOST1=&quot;192.168.6.2&quot; SERVERHOST2=&quot;192.168.7.2&quot;">
       <CONFIGURATIONS>
         <CONFIGURATION name="Debug" libraryPath="/usr/X11R6/lib/" isDebug="1" optimisation="1"
-                       targetName="LumatoneSetup"/>
+                       targetName="LumatoneSetup" defines="JUCE_MODAL_LOOPS_PERMITTED=1"/>

```

Comment:
I don't know exactly why modal loops need to be permitted but it solves a compilation problem: https://forum.juce.com/t/juce-filechooser-has-no-member-browseforfiletoopen/48535

Reply:
As far as I understand this line is no longer required if the code makes use of async modal dialogs?

Reply:
Yeah that was a JUCE 6 or 6.1 change. I switched to the async model in my development branch so we can do away with that there.